### ISSUE TITLE
SecretServiceKeychainAccess doesn't work with the latest secret-service library

### ISSUE THREAD

**Issue by SailReal** [cryptomator/integrations-linux#13]
After #12 is fixed, we have a new issue here: While startup in [SecretServiceKeychainAccess#isLocked](https://github.com/cryptomator/integrations-linux/blob/0e13a3fb552ac35e5d29bc12b5513a1a9a0226f2/src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java#L27-L34) we check if the `keyring` is locked. As this `keyring` is instantiated in an try-with statement, it will be closed after leaving the scope. In [SimpleCollection#disconnectSynchronized](https://github.com/swiesend/secret-service/blob/71bbddcc833a6712e0d8c14e326678abdb798456/src/main/java/org/freedesktop/secret/simple/SimpleCollection.java#L211) we set the static variable `connection` to `null`. When selecting a vault [SecretServiceKeychainAccess#isLocked](https://github.com/cryptomator/integrations-linux/blob/0e13a3fb552ac35e5d29bc12b5513a1a9a0226f2/src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java#L27-L34)  is called again or when clicking unlock, [SecretServiceKeychainAccess#loadPassphrase](https://github.com/cryptomator/integrations-linux/blob/3ce4dce7918a2f1e83f7ad5167675000e13e543c/src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java#L51-L62) is executed, it reuses the static variable `connection` which means `getConnection` isn't executed again. `SimpleCollection` calls the `init` method which calls `isAvailable` which checks `connection != null` and fails as we set `connection` to null.

**Comment by swiesend** [comment]
I introduced this bug, when I wanted to fix https://github.com/swiesend/secret-service/issues/26. I think it is better to give manual control for the `disconnect` of the DBus connection in order to separate the `AutoClose` behavior from `SimpleCollection` instances, from the global DBus connection which gets reused as it is declared within the static scope (in order to make `isAvailable()` work).

Will publish a new version soon.

**Comment by swiesend** [comment]
I released [1.7.0](https://github.com/swiesend/secret-service/blob/v1.7.0/CHANGELOG.md) to address the issues from cryptomator/integrations-linux#13 and cryptomator/integrations-linux#12.

Please let me know if there are problems. I built 
https://github.com/cryptomator/integrations-linux/tree/develop and https://github.com/cryptomator/cryptomator/tree/master with my 1.7.0 version and it didn't hang-up when the default-collection was locked.
